python/ray/autoscaler/_private/util.py

```diff
@@ -401,7 +401,7 @@ def with_envs(cmds: List[str], kv: Dict[str, str]) -> str:
     for cmd in cmds:
         kv_str = ""
         for k, v in kv.items():
-            kv_str += f"{k}={v} "
+            kv_str += f"export {k}={v}; "
```

Comment:
Add a comment why we need export. Maybe just link to this issue

Reply:
is link necessary (lol? again)
i thought with `;` is sufficient.